packages/components/nodes/chains/SqlDatabaseChain/SqlDatabaseChain.ts

```diff
@@ -81,11 +96,18 @@ class SqlDatabaseChain_Chains implements INode {
     }
 }
 
-const getSQLDBChain = async (databaseType: 'sqlite', dbFilePath: string, llm: BaseLanguageModel) => {
-    const datasource = new DataSource({
-        type: databaseType,
-        database: dbFilePath
-    })
+const getSQLDBChain = async (databaseType: DatabaseType, url: string, llm: BaseLanguageModel) => {
+    const datasource = new DataSource(
+        databaseType === 'sqlite'
+            ? {
+                  type: databaseType,
+                  database: url
+              }
+            : ({
+                  type: databaseType,
+                  url: url
```

Comment:
for other database types apart from `sqlite`, don't we need other configuration like `port`, `host` and others?
[example](https://github.com/FlowiseAI/Flowise/blob/main/packages/server/src/DataSource.ts)

Reply:
I think person is trying to encapsulate everything in connection string itself